lib/galaxy/jobs/runners/slurm.py

```diff
@@ -88,6 +90,8 @@ def __get_jobinfo():
                     return
             if drmaa_state == self.drmaa_job_states.DONE:
                 with open(ajs.error_file, 'r+') as f:
+                    if os.path.getsize(ajs.error_file) > SLURM_MEMORY_LIMIT_SCAN_SIZE:
+                        f.seek(-SLURM_MEMORY_LIMIT_SCAN_SIZE, os.SEEK_END)
```

Comment:
I think that this is now overwriting the initial part of the stderr file with the latest 16 MB, messing up its content.